### ISSUE TITLE
Faulty core fixes: ?> indented incorrectly after adding curly braces

### ISSUE THREAD

**Issue by pento** [WordPress-Coding-Standards/WordPress-Coding-Standards#1190]
Given the following test case:

```php
<?php
// Do something here...
?>

<?php if ( $foo ) :
	if ( $bar )
		$baz = 'cat'; ?>

	<div/>
<?php endif;
```

The `?>` at the end of the `$baz` line indented too far after the curly braces are added to that inner `if`. If the curly braces already exist, it indents correctly.

Core ref: src/wp-admin/index.php

Tested with PHPCS `master`, WPCS `develop`.

**Comment by jrfnl** [comment]
Ok, I've had a look at this and am not sure whether there is anything we can do about this really (other than running the sniffs without the `Squiz.PHP.EmbeddedPhp` sniff in a first run and with it in a second run).

The sniffs are registered in a non-conflicting order, so that's not it

What's happening is that - based on the above snippet -:
* PHPCS walks the tokens and triggers each sniff registered to a token in turn.
* The `<?php` on line 5 trigger the `EmbeddedPhp` sniff - which then moves the `?>` to the next line and indents it to be aligned with the first content of the line above it, i.e. `$baz = ...`.
* Then the `if` in `if ( $bar )` on line 6 triggers the `Generic.ControlStructures.InlineControlStructure` sniff which adds the braces.
* Now the line above the `?>` is indented one less tab as the brace has been placed, but as PHP open/close tags are used to _determine_ scope indent, the scope indent sniff cannot fix this - chicken-egg.

Essentially, this is the same issue as #1121 

These instances would all need to be inspected and potentially fixed before the big core run.

You could try and do a run with just `phpcs --sniffs=Squiz.PHP.EmbeddedPhp` to just see the errors thrown up by that sniff to get some indication of all the code affected, but that won't paint the complete picture as - like in the above example - other sniffs may reformat embedded PHP snippets which will influence what the `EmbeddedPhp` sniff throws up in subsequent fixer loops.

You basically would need to search for any embedded PHP snippets which contain more than one PHP statement, as those are the problematic ones.

**Comment by pento** [comment]
👍🏻 Thanks for the explanation!
